lib/Echidna/Types/Campaign.hs

```diff
@@ -211,9 +211,14 @@ defaultSymExecAskSMTIters :: Integer
 defaultSymExecAskSMTIters = 1
 
 -- | Get number of fuzzing workers (doesn't include sym exec worker)
--- Defaults to 1 if set to Nothing
+-- Defaults to (N-1) if set to Nothing, where N is Haskell's -N value,
+-- usually the number of cores, clamped between 1 and 3.
 getNFuzzWorkers :: CampaignConf -> Int
-getNFuzzWorkers conf = fromIntegral (fromMaybe 1 (conf.workers))
+getNFuzzWorkers conf = maybe defaultN fromIntegral conf.workers
+  where
+    n = numCapabilities
```

Comment:
is it the same as `getNumCapabilities`?

Reply:
yes, but this is ["pure"](https://hackage.haskell.org/package/ghc-internal-9.1001.0/docs/src/GHC.Internal.Conc.Sync.html#numCapabilities), `getNumCapabilities` is IO